examples/applications/plot_model_complexity_influence.py

```diff
@@ -72,12 +71,14 @@ def generate_data(case):
     """Generate regression/classification data."""
     if case == "regression":
         X, y = datasets.load_diabetes(return_X_y=True)
+        train_size = 0.8
     elif case == "classification":
         X, y = datasets.fetch_20newsgroups_vectorized(subset="all", return_X_y=True)
-    X, y = shuffle(X, y)
-    offset = int(X.shape[0] * 0.8)
-    X_train, y_train = X[:offset], y[:offset]
-    X_test, y_test = X[offset:], y[offset:]
+        train_size = 0.4
```

Comment:
```suggestion
        train_size = 0.4  # to make the example run faster
```

Reply:
The speed was not an issue. I just thought changing too many params was not an option.  I'll try to find optimal `n_samples `for other regressors.